onnx/reference/custom_element_types.py

```diff
@@ -1,14 +1,20 @@
 # Copyright (c) ONNX Project Contributors
-#
+
 # SPDX-License-Identifier: Apache-2.0
 from __future__ import annotations
 
-import numpy as np
+import warnings
+
+from onnx.custom_element_types import bfloat16  # noqa: F401
+from onnx.custom_element_types import float8e4m3fn  # noqa: F401
+from onnx.custom_element_types import float8e4m3fnuz  # noqa: F401
+from onnx.custom_element_types import float8e5m2  # noqa: F401
+from onnx.custom_element_types import float8e5m2fnuz  # noqa: F401
+from onnx.custom_element_types import int4  # noqa: F401
+from onnx.custom_element_types import uint4  # noqa: F401
 
-bfloat16 = np.dtype((np.uint16, {"bfloat16": (np.uint16, 0)}))
-float8e4m3fn = np.dtype((np.uint8, {"e4m3fn": (np.uint8, 0)}))
-float8e4m3fnuz = np.dtype((np.uint8, {"e4m3fnuz": (np.uint8, 0)}))
-float8e5m2 = np.dtype((np.uint8, {"e5m2": (np.uint8, 0)}))
-float8e5m2fnuz = np.dtype((np.uint8, {"e5m2fnuz": (np.uint8, 0)}))
-uint4 = np.dtype((np.uint8, {"uint4": (np.uint8, 0)}))
-int4 = np.dtype((np.int8, {"int4": (np.int8, 0)}))
+warnings.warn(
+    "This file was moved to onnx.custom_element_types. Importing this module will fail in onnx>=1.19.0.",
+    category=DeprecationWarning,
```

Comment:
Use future warning for deprecations. DeprecationWarnings are reserved for the Python language deprecations

Reply:
done